packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -46,6 +63,49 @@ export class BotConfig {
         botonic_cli_version: botonicCliVersion,
       },
       packages,
+      tools: configLoaded.tools,
+      payloads: configLoaded.payloads,
+      webviews: configLoaded.webviews,
+    }
+  }
+  static async getTools(appDirectory: string) {
+    try {
+      const botConfigPath = path.join(appDirectory, '.src/bot-config.ts')
+      const output = await this.getOutputByCommand(
+        `npx ts-node ${botConfigPath}`
+      )
+      console.log('output', output)
+    } catch (err: any) {
+      console.error(`Error: ${err.message}`)
+    }
+    return []
+  }
```

Comment:
Doubt: Is this function used at all? If it is not used you can remove it.

Reply:
removed